libs/zedUpload/datastore_aws.go

```diff
@@ -87,6 +87,12 @@ func (ep *AwsTransportMethod) WithSrcIPAndProxySelection(localAddr net.IP,
 	return nil
 }
 
+// WithSrcIPAndSkipVerify skip verify local datastore
+func (ep *AwsTransportMethod) WithSrcIPAndSkipVerify(localAddr net.IP) error {
+	return nil
```

Comment:
Shouldn't this return 	return fmt.Errorf("not supported")?

Reply:
Or do what you do for https?

Reply:
We can, but no one checks those return values on all those routines

Reply:
Actually they should all do what you do for https AFAICT.